CHANGELOG.unreleased.md

```diff
@@ -14,6 +14,7 @@
 ### Bug fixes
 
 - [Metadata backup] Missing XAPIs should trigger a failure job [#4281](https://github.com/vatesfr/xen-orchestra/issues/4281) (PR [#4283](https://github.com/vatesfr/xen-orchestra/pull/4283))
+- [New VM] Chrome should display tooltip on disabled form elements [#4304](https://github.com/vatesfr/xen-orchestra/issues/4304) (PR [#4309](https://github.com/vatesfr/xen-orchestra/pull/4309))
```

Comment:
Display tooltip on disabled form elements in Google Chrome ?

Reply:
It's a bug description, so we have to write what is expected.
I think `Tooltip should be displayed on disabled elements`
@julien-f what do you think ?

Reply:
I have no hard opinions on the form, but maybe it could be more user oriented: `Fix tooltips not displayed on disabled elements in some browsers (e.g. Google Chrome)`